deebot_client/messages/xml/stats.py

```diff
@@ -0,0 +1,15 @@
+"""Stats messages."""
+from deebot_client.event_bus import EventBus
+from deebot_client.message import HandlingResult, MessageBodyDataDict
+
+
+class ReportStats(MessageBodyDataDict):
+    """Report stats message."""
+
+    name = "GetReportStats"
```

Comment:
This looks like a command?
The difference between commands and messages is that we can only request/execute commands. Messages are send by the robot on the `atr` topic